corehq/apps/api/user_updates.py

```diff
@@ -30,6 +33,9 @@ def update(user, field, value, user_change_logger=None):
         'user_data': _update_user_data,
     }.get(field)
 
+    if not update_fn:
+        raise UnknownFieldException
```

Comment:
I know this exception will usually be caught and transformed to a human-readable error, but it still might be nice to show the field name in the exception message in case it is ever raised in a context where it is not handled.
```suggestion
        raise UnknownFieldException(field)
```

Reply:
See bd780709adcf66f177ca3ad73d276e7321b862eb